### ISSUE TITLE
Node/JS Beginner Study Group

### ISSUE THREAD

**Issue by max-mapper** [nodeschool/pdx#3]
I'd like to start a small JavaScript and Node study group in PDX for beginners.

The Study Group format is inspired by https://mozillascience.github.io/studyGroupHandbook/.

For the first one, if you want to join the Node + JavaScript study group email me at **mogden at gmail dot com** if you can commit to showing up to at least the first one. Once we have 4 people confirmed we can have the first one! I'd like to limit it to 4 people at first just to make scheduling a time easier.

For the first one how about **Tuesday, Nov 10th, 6:30pm** at a coffee shop in SE? Exact location details can be exchanged through email. If this doesn't work we can try and find a better time for people, but weeknight evenings is probably the easiest.

If you are an absolute beginner or consider yourself a "newbie" then this is the group for you. Otherwise maybe we can form another intermediate group later.

To summarize:
- If you're interested in joining the study group, email me!
- This study group will be for beginners
- Let me know in the email if  **Tuesday, Nov 10th, 6:30pm** does or does not work for you

**Comment by bcomnes** [comment]
Did the first meeting happen?  How did it go? Pics?

Are people interested in tracking curriculum topics, links, etc similar to https://github.com/cyberwizardinstitute/workshops

**Comment by mootpt** [comment]
@maxogden reached out with you via email. Did this first meeting happen? I am interested in joining the next study group.

**Comment by max-mapper** [comment]
@mootpt cool thanks i'll let you know
